### PR TITLE
SUBMARINE-306. Add some steps of creating a image of mini-submarine

### PR DESCRIPTION
**Pull request by Eroschang**
### What is this PR for?
Add some steps to build a personal image of mini-submarine.

### What type of PR is it?
Improvement

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-306

### How should this be tested?
https://travis-ci.org/apache/submarine/builds/618603483

### Questions:
* Does the licenses files need update? Yes/No
* Is there breaking changes for older versions? Yes/No
* Does this needs documentation? Yes/No

### CODE REVIEW

File: dev-support/mini-submarine/README.md [hunk 1]
```diff
@@ -31,8 +31,21 @@ docker pull hadoopsubmarine/mini-submarine:0.3.0-SNAPSHOT
...
+```
+
+2.Build Submarine
+```
+cd ./submarine
+mvn clean install package -DskipTests
+```
 
+3.Build image of mini-submarine
+> You can pre-download these three kind of compressed file in the same file for building : zookeeper-3.4.14.tar.gz , hadoop-2.9.2.tar.gz , spark-2.4.4-bin-hadoop2.7.tgz
```
Comment (liuxunorg) [inline dev-support/mini-submarine/README.md]: Download in advance, where can I download it?

**Comment by liuxunorg** [comment]
@Eroschang Please change PR title to `SUBMARINE-306. Add some steps of creating a image of mini-submarine`

**Comment by liuxunorg** [comment]
@Eroschang Thanks for your contribution.
Will merge if no more comment.

**Comment by Eroschang** [comment]
@liuxunorg OK , thanks a lot for your help.

**Pull request closed without merging**